prbt_hardware_support/test/unittests/unittest_pilz_modbus_client.cpp

```diff
@@ -439,29 +439,16 @@ class RegisterBuffer
  * @brief Tests that the frequency with which the registers are read,
  * can be changed.
  *
- * This test does NOT precisly check (tolerance ca. 10Hz) if the exact
+ * This test does NOT precisly check (tolerance ca. 10%) if the exact
  * frequency could be set. It just checks that the frequency was
  * changed.
  *
  *
  * Test Sequence:
- * - 1. Choose low read frequency (f1). The last received "modbus_read" message
- *      is stored in a buffer.
- * - 2. Choose "modbus_read" topic check frequency f2, with f2 slightly bigger than f1.
- * - 3. Start a loop (running exactly for \#f1 iterations), which checks with
- *      frequency f2 the last received "modbus_read" message.
- * - 4. Stop all loops and the PilzModbusClient.
+ * - 1. Choose a read frequency. Measure the number of register accesses within ~3 seconds.
```

Comment:
In order to check, if a frequency can be changed, the whole test case should be performed twice with different read frequencies.

Reply:
Can we reduce to ~1 sec?

Reply:
Done. Done.